app/liquid/liquid_renderer.rb

```diff
@@ -43,6 +44,15 @@ def images
 
   private
 
+  def set_locale
+    begin
+      if @page.language.present? && @page.language.code.present?
```

Comment:
I think the `.code.present?` check is redundant here, because `code` should be a required value on an instance of `Language`.